File: programme/reverse-of-a-number/reverse-of-number.c
// Write a programmeto print the reverse of a number
#include <stdio.h>

int main()
{
    int a, n;
    printf("Input  : ");
    scanf("%d", &a);
    printf("Output : ");
    while (a > 0)
    {
        n = a % 10;
        printf("%d", n);
        a /= 10;
    }
    printf("\n");
    return 0;
}

File: programme/reverse-of-a-number/reverse-of-number.cs
// Write a programmeto print reverse of a number
using System;
using System.Linq;

public class Reverse
{
    static void Main(string[] args)
    {
        int Reverse(int input)
        {
            return int.Parse(new string(input.ToString()
                .Reverse()
                .ToArray()
            ));
        }

        // Input number
        Console.Write("Enter numbers : ");
        int input = int.Parse(Console.ReadLine());


        // output reversed number
        Console.WriteLine("Reversed : " + Reverse(input));
    }
}
